### ISSUE TITLE
Correcting wording and details for interface text.

### ISSUE THREAD

**Issue by Calyp** [freedombox/Plinth#375]
During translation to Swedish I have noticed several discrepancies in the original English text. 

Item No 24:
Hostname is the local name by which other machines on the local network reach your machine.  It must start and end with an alphabet or a digit and have as interior characters only alphabets, digits and hyphens.  Total length must be 63 characters or less."

>Suggested correction and re-wording: 
"Hostname is the local name by which other devices on the local network can reach your FreedomBox. It may only contain letters, numbers and hyphens, and can not start or end with a hyphen. It must be shorter than 64 charachters."

Item No 27:
"Domain name is the global name by which other machines on the Internet can reach you.  It must consist of labels separated by dots.  Each label must start and end with an alphabet or a digit and have as interior characters only alphabets, digits and hyphens.  Length of each label must be 63 characters or less.  Total length of domain name must be 253 characters or less."

>Suggested correction and re-wording:
"Domain name is the global name by which other devices on the Internet can reach your FreedomBox. It must consist of labels separated by dots. Each label may only contain letters, numbers and hyphens, and can not start or end with a hyphen. Each label must be shorter than 64 characters, and the domain name must be shorter than 254 characters" 

Item No 76:
"Please do not enter a URL here (like "https://example.com/") but only the hostname of the GnuDIP server (like "example.pcom")."

>Change .pcom to .com in last example:
"(like "example.pcom")." ---> "(like "example.com")."
NOTE: This was missed in some translations. I checked all translations and corrected those that missed this. (French, Norwegian, Danish and Turkish) 

Item No 77:
"The public domain name you want use to reach your box."

>For concistency, refer to the FreedomBox device as the FreedomBox, not "your box"
---> "The public domain name you want use to reach your FreedomBox."

Item No 81:
"Optional Value. If your FreedomBox is not connected directly to the Internet (i.e. connected to a NAT router) this URL is used to figure out the real Internet IP. The URL should simply return the IP wherethe client comes from. Example: http://myip.datasystems24.de"

>Missing space:
"...The URL should simply return the IP wherethe ..." --> "where the"
For consistency examples should be within parenthesis:
"Example: http://myip.datasystems24.de"  ---> "(example: http://myip.datasystems24.de)"

Items No 88, 89 & 92:
88 "accept all SSL certificates"
89 "use HTTP basic authentication"
92 "show password"

>For consistency, those should all start with capital letter:
--> "Accept all SSL certificates"
--> "Use HTTP basic authentication"
--> "Show password"

Item No 100:
"DynamicDNS client"

>Missing space:
"DynamicDNS client" ---> "Dynamic DNS client"

Item 102:
"The solution is to assign a DNS name to your IP address and update the DNS name every time your IP is changed by your Internet provider. Dynamic DNS allows you to push your current public IP address to an <a href='http://gnudip2.sourceforge.net/' target='_blank'> gnudip </a> server. Afterwards the Server will assign your DNS name with the new IP and if someone from the internet asks for your DNS name he will get your current IP answered."

>Internet is always spelled with capital "I" (last sentence):
"...if someone from the internet..." ---> "...if someone from the Internet..."

Item No 104:
"If your freedombox is connected behind some NAT router, don't forget to add portforwarding (i.e. forward some standard ports like 80 and 443) to your freedombox device."

>For consitency and correctness, FreedomBox should be written with capital F and B, and also, the parenthesis in unessecarily messy, suggested text including removed redundancy and added missing space:
 ---> "If your FreedomBox is connected behind some NAT router, don't forget to add port forwarding. (i.e. forward some standard ports like 80 and 443)"

Additional Notes:
>The wording in general are littered with inconsistencies referring to the FreedomBox device as "The box", "your device", "your machine", "the freedombox", "the FreedomBox" and "the FreedomBox device".

**Comment by SunilMohanAdapa** [comment]
Fixed the items except for following comments:

> "Hostname is the local name by which ...

Fixed the first statement.  Kept the rest as the new sentence didn't seem like an significant improvement.

> "Domain name is the global name ...

Fixed the first statement. Kept the rest as the new sentence didn't seem like an significant improvement and lost some information about how domain names are combinations of labels.

> The wording in general are littered with inconsistencies referring to the FreedomBox device as "The box", "your device", "your machine", "the freedombox", "the FreedomBox" and "the FreedomBox device". 

I worked on fixing brand name usage in a lot of place but didn't work on this.  I could use some help.
